src/lib/expression/evaluation/like_matcher.cpp

```diff
@@ -86,20 +86,19 @@ LikeMatcher::AllPatternVariant LikeMatcher::pattern_string_to_pattern_variant(co
     return ContainsPattern{std::get<pmr_string>(tokens[1])};
 
   } else {
-    /**
-     * Pattern is either MultipleContainsPattern, e.g., '%hello%world%how%are%you%' or we fall back to
-     * using a regex matcher.
-     *
-     * A MultipleContainsPattern begins and ends with '%' and  contains only strings and '%'.
-     */
-
-    // Pick ContainsMultiple or Regex
-    auto pattern_is_contains_multiple = true;  // Set to false if tokens don't match %(, string, %)* pattern
-    auto strings = std::vector<pmr_string>{};  // arguments used for ContainsMultiple, if it gets used
-    auto expect_any_chars = true;              // If true, expect '%', if false, expect a string
+    // Pattern is either MultipleContainsPattern, e.g., 'hello%world%' or '%hello%world%how%are%you%'. If not, we fall
```

Comment:
I think an example for a string that would fall back to the regex matcher would be helpful.

Reply:
Changed to
```
// Pattern is either MultipleContainsPattern, e.g., 'hello%world%' or '%hello%world%how%are%you%'. If not, we fall
    // back to using a regex matcher (mostly patterns with single char wildcards).
```